### DISCUSSION TITLE
Won't opened if unicode characters in markdo file name.

### DISCUSSION THREAD

**Discussion by jeedy** [community/community#48250] [Other Feature Feedback, Questions, & Ideas] [answered]
![image](https://user-images.githubusercontent.com/2126502/220547546-1fb18146-365a-4d7e-9fe6-75428963e689.png)

**Comment by queenofcorgis** [comment, marked as answer]
Thanks for posting in the GitHub Community, @jeedy ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by Tonimanero123123** [reply]
My love

**Reply by Sherr75** [reply]
I need help getting a 2 years hacker out of my phone she and Johnie Avrett stole foot of money but has left all accounts in my name but has completaken control of my whole phone and accouh

**Comment by Tonimanero123123** [comment]
Lokalizacja telefonu
